### ISSUE TITLE
App is crashed at "CHSesame5Device#readHistoryCommand"

### ISSUE THREAD

**Issue by takao-akiyoshi** [CANDY-HOUSE/SesameSDK_iOS_with_DemoApp#71]
We faced this issue too many times from 2023/12/11 11:45:29 to 2023/12/11 13:27 (over 72 times!)
And we don't face this with same environment before yesterday.
Are there some update on your side??
![スクリーンショット 2023-12-11 13 44 37](https://github.com/CANDY-HOUSE/SesameSDK_iOS_with_DemoApp/assets/8392668/8bd128d2-4f37-4d96-84be-60d79de63fe1)

**Comment by takao-akiyoshi** [comment]
[NITS]
Runtime says
`#0	(null) in Swift runtime failure: Range requires lowerBound <= upperBound ()`

So, it needs to fix like as blow, right?
### AS IS
```
guard hisContent.count > originalTagCount else { return }
hisContent = originalTagCount.data + hisContent[1...originalTagCount].copyData
```

### TO BE
```
guard hisContent.count > originalTagCount,
      let content = hisContent[safeBound:1...Int(originalTagCount)]
else { return }
hisContent = originalTagCount.data + content.copyData
```

**Comment by Jerming** [comment]
Yes, you are correct. Will fix the problem soon. Thank you so much.
Sorry for this inconvenience.
